crates/extensions/c8y_mapper_ext/Cargo.toml

```diff
@@ -17,6 +17,7 @@ camino = { workspace = true }
 clock = { workspace = true }
 json-writer = { workspace = true }
 logged_command = { workspace = true }
+nanoid = { workspace = true }
```

Comment:
We can consider to move the generation of a unique id for a command in `MqttSchema` as this will have to be done for all command types.